src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -52,6 +62,43 @@ export function PharmacyCard(props: PharmacyProps) {
       styles={{ border: "none" }}
     />
   );
+  const generateRows = () => {
+    const rows: (string | number)[][] = [];
+    props.vaccineAvailabilities.forEach((availability) => {
+      const newRow = [
+        format(new Date(availability.date), "MMM d, y"),
+        availability.numberAvailable,
+      ];
+      rows.push(newRow);
+    });
+    return rows;
+  };
+  const dataTableMarkup = shouldShowSlots ? (
+    <DataTable
+      columnContentTypes={["text", "numeric"]}
+      headings={["Date", "Quantity"]}
+      rows={generateRows()}
+    />
+  ) : undefined;
+
+  const appointmentsAvailableMarkup = () => {
+    if (props.vaccineAvailabilities.length !== 0) {
+      return (
+        <Card.Section title="Dates">
+          <Button
+            primary
+            onClick={() => {
+              setShouldShowSlots(!shouldShowSlots);
+            }}
+          >
+            See Slots
```

Comment:
You need to toggle this text between "Show" and "Hide"

Now that we have localization rolled in, we should update the locale files... We'll also need a process to get the outstanding French translations done and rolled into the codebae.